modules/administration-guide/examples/image_puller/example_image_puller.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: che.eclipse.org/v1alpha1
-kind: KubernetesImagePuller
+kind: {image-puller-cr-name}
 metadata:
   name: image-puller
   namespace: <namespace you installed the image puller in>
 spec:
-  configMapName: k8s-image-puller
-  daemonsetName: k8s-image-puller
-  deploymentName: kubernetes-image-puller
+  configMapName: {image-puller-deployment-name}
```

Comment:
Have all these been verified? `k8s-image-puller` and `kubernetes-image-puller` are two distinct names...

Reply:
I believe this is a mistake or a leftover: in the other locations `kubernetes-image-puller` is used for `daemonsetName` and  `deploymentName`.